hibernation-setup-tool.c

```diff
@@ -1707,6 +1707,31 @@ static void ensure_systemd_hooks_are_set_up(void)
 
 int main(int argc, char *argv[])
 {
+    char *dest_dir = NULL;
```

Comment:
Will it be used anywhere currently? If not remove it for now?

Reply:
nvm - got the use case from this PR - https://github.com/microsoft/hibernation-setup-tool/pull/7